### DISCUSSION TITLE
Organização de Projetos

### DISCUSSION THREAD

**Discussion by reluiz** [balta-io/forum#527] [Tira Dúvidas] [answered]
Boa tarde! Concluí o curso Criando uma API Completa e já comecei a desenvolver a API para o nosso software. Comecei também a estudar o curso de Blazor Web Assembly para desenvolver o front-and. Gostaria de saber se separar as soluções para organizar os projetos seria uma boa prática. Atualmente tenho uma solução e nessa solução eu tenho apenas o projeto Server API que será publicado  no Azure. Pensei em criar uma outra solução que seria responsável pelos projetos do front-end, então teríamos duas soluções uma para o back-end e uma para o front-end. Pergunto isso por conta do Azure, para publicar lá o cenário menos caótico seria um Resources Group para gerenciar uma única solução com os projetos de back e front ou seria melhor também separar ficando um resource group para o back e outro para o front.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @reluiz, como vai?

É uma boa sim, mas não é obrigatório! Você pode criar tudo no mesmo projeto ou manter separado, depende do que faz mais sentido para o seu projeto.

Se for estudos apenas, faz os dois para testar, assim você já tem uma ideia. Inclusive varia com deploys separados e juntos.
